esprima.js

```diff
@@ -6173,7 +6186,11 @@ parseYieldExpression: true, parseAwaitExpression: true
 
     function parseClassImplements() {
         var id, implemented = [], marker, typeParameters;
-        expectContextualKeyword('implements');
+        if (strict) {
+            expectKeyword('implements');
```

Comment:
Woah there esprima...[implements is a future reserved word](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-future-reserved-words), right?

This isn't the fault of the PR. I was a little sketched out by `static` being a keyword ([it's not!](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-keywords)) but this seems even more off.